docs/conf/portland/2018/sponsors/prospectus.rst

```diff
@@ -229,3 +229,22 @@ of our expenses must be paid up front.
 
 .. _ticket: https://ti.to/writethedocs/write-the-docs-portland-2018/
 .. _tickets: https://ti.to/writethedocs/write-the-docs-portland-2018/
+
+Frequently Asked Questions
+==========================
+
+What are the booth sizes? What is included?
+-------------------------------------------
+
+The small sponsorship booth is a 6' table, and the large is two 6' tables. These are not standard conference expo halls, but in the main conference venue, so please don't bring anything that will need to be hung or expand more than 2 feet beyond the edge of your area. Wifi & Power will be provided.
+
+What are the dates that sponsors need to know about?
+----------------------------------------------------
+
+* PRIOR: Two weeks prior to the event, you are able to start shipping items for the conference. Please inquire about the address to ship things to.
```

Comment:
Surely we send them the address upon sponsorsip? But probably doesn't hurt to restate.